.github/workflows/next-build-image.yaml

```diff
@@ -12,12 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: Build Next Image
+name: Build Next and Tag Image
 
 on:
+  schedule:
+    # run at 3:18 UTC every day
+    - cron: "18 3 * * *"
```

Comment:
I'm confused why you want to switch from "on every push to main" to "once a day"... are we trying to reduce noise ? server usage? data centre heat causing global warming?

Reply:
If you do "on every push to main" than every time, there is a new commit on main, the running job gets canceled, and a new one is trigerred. Which makes sense because otherwise you are building something that is already out of date. 

To complete a full multiplatform image build, which typically takes over 5 hours, you’d need at least five consecutive hours without merging any pull requests (PRs) into main.
During regular workdays, with multiple PRs being merged daily, this job will likely get triggered and canceled repeatedly until nighttime when there’s less activity.

Reply:
FIVE HOURS? What's it doing, compiling every single dependent package from sources??

Reply:
also you can turn off the concurrency settings to cancel in-progress builds if you don't want things to be cancelled and started over:

https://github.com/redhat-developer/rhdh/blob/main/.github/workflows/next-build-image.yaml#L24

```
concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}
  cancel-in-progress: true
```

Reply:
yes, but if we don't cancel it than we are building something that is  is never going to be used